templates/house_create/house_create.html

```diff
@@ -0,0 +1,5 @@
+{% extends "base.html" %}{% load static %}{%block content%}
+
+house creation
+
+{% endblock content %}
```

Comment:
Missing empty line

Reply:
> Missing empty line

added an empty line at the end of the file 
thanks👍